inst/qml/msaGaugeRR.qml

```diff
@@ -532,4 +532,164 @@ Form
 			}
 		}
 	}
+
+	Section
+	{
+		title: 									qsTr("Advanced Options")
+		columns:								1
+
+		Group
+		{
+			title:		qsTr("Tests for control charts")
+
+			DropDown
+			{
+				name:									"testSet"
+				label:									qsTr("Test set")
+				id: 									testSet
+				indexDefaultValue:						0
+				values: [
+					{ label: qsTr("JASP"), value: "jaspDefault"},
+					{ label: qsTr("Nelson laws"), value: "nelsonLaws"},
+					{ label: qsTr("Western Electric rules"), value: "westernElectric"},
+					{ label: qsTr("Custom selection"), value: "custom"}
+				]
+			}
+
+
+			CheckBox
+			{
+				name: 								"rule1"
+				label: 								qsTr("Points outside of control limits")
+				checked:							true
+				enabled:							testSet.currentValue == "custom"
```

Comment:
I would do visible instead of enabled for all these. Also applies to the other qml files that do the same